lib/Doctrine/Persistence/Mapping/Driver/SymfonyFileLocator.php

```diff
@@ -55,30 +59,30 @@ class SymfonyFileLocator implements FileLocator
     private $nsSeparator;
 
     /**
-     * @param string[]    $prefixes
-     * @param string|null $fileExtension
-     * @param string      $nsSeparator   String which would be used when converting FQCN to filename and vice versa. Should not be empty
+     * @param string[] $prefixes
+     * @param string   $nsSeparator String which would be used when converting FQCN to filename and vice versa. Should not be empty
      */
-    public function __construct(array $prefixes, $fileExtension = null, $nsSeparator = '.')
-    {
+    public function __construct(
+        array $prefixes,
+        ?string $fileExtension = null,
+        ?string $nsSeparator = '.'
```

Comment:
Any reason why we made this nullable?

Reply:
Hmm, let me double check that and see if I can get rid of it.